tests/dummy/app/templates/map.hbs

```diff
@@ -20,6 +20,22 @@
           <i class="info circle icon"></i>
         </button>
       </div>
+      <div class='ui compact menu' style='border-width:0px;border-image-width:0px;box-shadow: rgba(0, 0, 0, 0) 0px 0px 0px 1px inset; hight:36px;'>
```

Comment:
I think we should create flexberry-mapmenu for this.
I have'nt time and skills to do it